### PR TITLE
Adding a new pin for Dearborn, MI

### LINKED ISSUE #163: Adding a pin for Dearborn, MI USA
## Adding <your-city> to the map

Hi everyone! This issue is to track the addition of Dearborn, MI to the map, which can be found at:

- Longitude: 42.315853
- Latitude: -83.217089

### PR DESCRIPTION
**Pull request by scwestfall**
Code for a pin in Dearborn, MI was added as requested by @githubteacher 
This change closes #163

### CODE REVIEW

File: scwestfall-dearborn-mi.topojson [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{ 
+  "type": "Feature", 
+  "geometry": {
+               "type": "Point", 
+               "coordinates": [42.315853, -83.217089]
```
Comment (ekxxi) [inline scwestfall-dearborn-mi.topojson]: Hmm... weird... I just noticed this. I always thought map coordinates were expressed in lat-long but this exercise reverses them. Even OpenStreetMap, which (presumably) powers MapBox, [expresses the coordinates in lat-long](https://www.openstreetmap.org/?mlat=42.314444&mlon=-83.213333&zoom=12#map=12/42.3144/-83.2133) as expected. Maybe this was done on purpose to see if we're paying attention?

Comment (hectorsector) [inline scwestfall-dearborn-mi.topojson]: It wasn't done on purpose, @ekxxi, but it is the default behavior so we didn't mess with it. In the GeoJSON implementation they do have to be flipped though.

**Comment by hollenberry** [comment]
Hey @scwestfall! This looks good to :ship:! Feel free to do so when you're ready. :smile:

**Comment by mossimo88** [comment]
Hello! 

I am adding myself to Dearborn! 

Thanks

**Comment by crichID** [comment]
@scwestfall this looks good to merge! Please feel free to click the big green button above to Merge pull request.

If you haven't merged by 11/15/2016, a member of the GitHub training team will merge it for you 😉

**Pull request merged**